### PR TITLE
fixing post partum language in review error and raw application

### PR DESCRIPTION
**Pull request by erubinst**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/n/projects/2640057/stories/185060658

# A brief description of the changes

Current behavior: Text does not match "the last year" wording in review, error message, and raw application

New behavior: Text will match other questions page for post partum language

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: POST_PARTUM_PERIOD_ONE_YEAR_IS_ENABLED

- [ ] DC
- [x] ME

### CODE REVIEW

**Review by scaustin34 — DISMISSED** [review]
LGTM!

File: features/financial_assistance/step_definitions/other_questions_page_steps.rb [hunk 1]
```diff
@@ -253,6 +253,10 @@
   choose('is_pregnant_no')
 end
 
+And(/the post partum language one year is not enabled/) do
```
Comment (mdkaraman) [inline features/financial_assistance/step_definitions/other_questions_page_steps.rb:256]: We have a helper set up that makes defining this step unnecessary.  We should be able to just add: 
```
And FAA post_partum_period_one_year feature is enabled
```
to the other_questions_pregnancy.feature file.  

Check out features/support/worlds/resource_registry_world.rb for the implementation details.

File: features/financial_assistance/other_questions_pregnancy.feature [hunk 2]
```diff
@@ -25,24 +25,28 @@ Feature: Start a new Financial Assistance Application and answers questions on O
 
   Scenario: Pregnancy response within 60 days Yes with information filled and submitted
     Given the user answers no to being pregnant
+    And the post partum language one year is not enabled
```
Comment (mdkaraman) [inline features/financial_assistance/other_questions_pregnancy.feature:28]: These lines can be replaced with:
```
And FAA post_partum_period_one_year feature is enabled
``` 
Check out the comment in the steps file for details.

**Review by mdkaraman — APPROVED** [review]
Looks good!

**Review by scaustin34 — APPROVED** [review]
LGTM!

**Pull request merged**